allennlp/nn/decoding/decoder_trainers/expected_risk_minimization.py

```diff
@@ -82,10 +88,39 @@ def _get_finished_states(self,
                 else:
                     next_states.append(next_state)
 
-            states = self._prune_beam(next_states)
+            states = self._prune_beam(next_states, self._beam_size, False)
```

Comment:
Minor readability nit: I find it easier to read when magic arguments are passed by name, so that I don't have to guess (or find the method definition) to know what `False` means here.  So, preference for `sort_states=False`.  Same with the `_prune_beam` invocation a couple of lines below.

Reply:
Done.